languageprocessing/src/main/java/commands_processing/CommandBuilder.java

```diff
@@ -0,0 +1,162 @@
+package commands_processing;
+
+import grammar.LanguageEnums.*;
+import grammar.LanguageMaps;
+import grammar.PhrasalVerb;
+import grammar.PhrasalVerbChecker;
+
+public class CommandBuilder {
+    private final LanguageMaps lMap;
+    CommandList cmList;
```

Comment:
cmList and pvChecker have no access modifiers. if this is not intentional then I it's probably best to change it